spec/margin.md

```diff
@@ -86,6 +84,18 @@ Deposits funds into an obligation account to be used to open a leveraged trade.
 
 - Can only be called by the trader.
 
+    ///   0. `[]` Deposit reserve account.
+    ///   1. `[]` Borrow reserve account.
+    ///   2. `[writable]` Obligation
+    ///   3. `[writable]` Obligation token mint
+    ///   4. `[writable]` Obligation token output
+    ///   5. `[]` Obligation token owner
+    ///   6. `[]` Lending market account.
+    ///   7. `[]` Derived lending market authority.
+
+    let deposit_reserve_info = next_account_info(account_info_iter)?;
+    let borrow_reserve_info = next_account_info(account_info_iter)?;
```

Comment:
```suggestion
```

Reply:
oh whoops, I was working on something else and this got pushed. will revert.